### DISCUSSION TITLE
Album order

### DISCUSSION THREAD

**Discussion by sajlala** [jcorporation/myMPD#617] [Q&A] [answered]
Thank you for mympd, I love it.

I have recently upgraded to the most recent release for the first time in 18 months. I use the album cover view and prior to the upgrade albums were ordered first by artist and then chronologically (I originally achieved this by saving my albums in folders with names that begin with the year of their release).
Since the upgrade the albums are appearing in the grid ordered first by artist and then alphabetically by album name.
I cannot work out how to get it back to how it was. Sorry for any ignorance, but do you have any advice?

**Comment by jcorporation** [comment, marked as answer]
Secondary ordering was undefined in old version and is undefined in current version.

The code behind the albumgrid has changed and that’s the cause why the secondary ordering has changed.

I think secondary ordering should be by albumname and not foldername.
